service-api/app/src/App/src/Service/Lpa/AddOlderLpa.php

```diff
@@ -92,14 +99,23 @@ public function validateRequest(string $userId, array $matchData): array
 
         // Find actor in LPA
         $resolvedActor = ($this->findActorInLpa)($lpaData, $matchData);
+
+        // Streamline lpa's sent to cleansing team that are registered after 2019 and when user details do not match
```

Comment:
```suggestion
        // We may want to turn off the ability for a user to have their case pushed to the cleansing 
        // team if they fail to match and have a "newer" older lpa. In which case they'll be told we 
        // can't find their LPA.
```

Just a bit more verbose so that we'll know what this is doing in the future.